src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -111,17 +138,19 @@ const throwInvalidPropComboError = (variant: Variant, status: Status) => {
  */
 export const ClickableStyle = React.forwardRef(
   <IComponent extends React.ElementType>(
-    {
+    props: ClickableStyleProps<IComponent>,
+    ref: React.ForwardedRef<HTMLElement>,
+  ) => {
+    const {
       as: Component = 'button',
       className,
       fullWidth,
       size = 'lg',
       status = 'brand',
       variant = 'secondary',
       ...other
-    }: ClickableStyleProps<IComponent>,
-    ref: React.ForwardedRef<HTMLElement>,
-  ) => {
+    } = props;
+
     if (
       !getPropCombinationIsValid(variant, status) &&
       process.env.NODE_ENV !== 'production'
```

Comment:
You can remove all the invalid prop console log stuff in here and the whole `ClickableStyle` test file now.

Reply:
Agreed. Any things building directly upon `ClickableStyle` should use the exported union to lock the props to the valid combinations. Removed!